WalletWasabi.Tests/UnitTests/Crypto/RandomTests.cs

```diff
@@ -31,8 +33,9 @@ public void BasicTests()
 			var pseudoSet = new HashSet<byte[]>();
 			var secureSet = new HashSet<byte[]>();
 			var count = 100;
-			var unsecureRandom = new UnsecureRandom();
-			using var secureRandom = new SecureRandom();
+			IWasabiRandom unsecureRandom = new UnsecureRandom();
+			using var secureRandomToDispose = new SecureRandom();
+			IWasabiRandom secureRandom = secureRandomToDispose;
```

Comment:
Personally, I would rename `IWasabiRandom.GetBytes(int length)` to something like `IWasabiRandom.GetBytesByLen(int length)` because it's confusing. 

Then it would be:

```csharp
IWasabiRandom unsecureRandom = new UnsecureRandom();
using SecureRandom secureRandom = new SecureRandom();

for (int i = 0; i < count; i++)
{
	pseudoSet.Add(unsecureRandom.GetByLength(10));
	secureSet.Add((secureRandom as IWasabiRandom).GetByLength(10));
}
```

Reply:
`length` is the name of the parameter, so `GetBytesByLen` naming would be redundant. I agree it's confusing and it bothers me too, but the interface comes from a library project.